### PR TITLE
Add expand verb

### PR DESCRIPTION
**Pull request by PietJankbal**
See issue https://github.com/Winetricks/winetricks/issues/1463

Usefull verb to extract files/dlls from a .msu package.

Regards

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -10037,6 +10047,48 @@ load_dirac()
...
+    w_warn "Do not set expand.exe and cabinet.dll to native in winecfg, it will break lots of installers!"
+
+    for i in x86_microsoft-windows-basic-misc-tools_31bf3856ad364e35_6.0.6001.18000_none_175cb770bf6b8f77/expand.exe \
+        x86_microsoft-windows-deltapackageexpander_31bf3856ad364e35_6.0.6001.18000_none_69de962f3507db4b/dpx.dll \
+        x86_microsoft-windows-cabinet_31bf3856ad364e35_6.0.6001.18000_none_373f511ce1ebb446/cabinet.dll \
+        x86_microsoft-windows-servicingstack_31bf3856ad364e35_6.0.6001.18000_none_095f6148c74a7a64/msdelta.dll; do
+
+        helper_winvista "$i"
+        w_try cp "$W_TMP/$i" "$W_SYSTEM32_DLLS/"
+
```
Comment (austin987) [inline src/winetricks]: Remove the trailing newline.

Do any of these dlls need to be registered?

File: src/winetricks [hunk 2]
```diff
@@ -10037,6 +10047,48 @@ load_dirac()
...
+        x86_microsoft-windows-cabinet_31bf3856ad364e35_6.0.6001.18000_none_373f511ce1ebb446/cabinet.dll \
+        x86_microsoft-windows-servicingstack_31bf3856ad364e35_6.0.6001.18000_none_095f6148c74a7a64/msdelta.dll; do
+
+        helper_winvista "$i"
+        w_try cp "$W_TMP/$i" "$W_SYSTEM32_DLLS/"
+        
+    done
+    
+    if [ "$W_ARCH" = "win64" ]; then
+
```
Comment (austin987) [inline src/winetricks]: Remove extra newline.

File: src/winetricks [hunk 3]
```diff
@@ -10037,6 +10047,48 @@ load_dirac()
...
+    
+    if [ "$W_ARCH" = "win64" ]; then
+    
+        for i in amd64_microsoft-windows-basic-misc-tools_31bf3856ad364e35_6.0.6001.18000_none_737b52f477c900ad/expand.exe \
+            amd64_microsoft-windows-deltapackageexpander_31bf3856ad364e35_6.0.6001.18000_none_c5fd31b2ed654c81/dpx.dll \
+            amd64_microsoft-windows-cabinet_31bf3856ad364e35_6.0.6001.18000_none_935deca09a49257c/cabinet.dll \
+            amd64_microsoft-windows-deltacompressionengine_31bf3856ad364e35_6.0.6001.18000_none_9c4b039c3e1c8af8/msdelta.dll; do
+
+            helper_winvista "$i"
+            w_try cp "$W_TMP/$i" "$W_SYSTEM64_DLLS/"
```
Comment (austin987) [inline src/winetricks]: Again, do they need to be registered?

File: src/winetricks [hunk 4]
```diff
@@ -10037,6 +10047,48 @@ load_dirac()
...
+        done
+    fi
+
+    #Do not set expand/cabinet to native, it breaks lots of installers (!), but to extract files it works ok. So give example to user
```
Comment (austin987) [inline src/winetricks]: Nitpick, but please put a space before `Do`.

File: src/winetricks [hunk 5]
```diff
@@ -10037,6 +10047,48 @@ load_dirac()
...
+    echo "Example of usage:
```
Comment (austin987) [inline src/winetricks]: `echo` won't display to GUI users; use `w_warn`

**Comment by PietJankbal** [comment]
So i messed it up again apparently, i deleted my local repository but now there`s nothing to squash anymore. I close this one and try again. Sorry for the noise

**Pull request closed without merging**